### PR TITLE
Supporting 8.1 (Remove deprecated interface) + Update and fix phpunit test

### PR DESCRIPTION
**Pull request by denis-rolling-scopes**

### CODE REVIEW

**Comment by dannyvw** [comment]
@Nyholm @sroze do you have any chance to look at this and create a new release?

**Comment by joel-loycom** [comment]
This is also blocking us, can it be merged?

**Comment by denis-rolling-scopes** [comment]
@dannyvw @joel-loycom you can use https://packagist.org/packages/autoprotect-group/messenger-enqueue-transport
it's a fork that supports 8.1

**Comment by qkdreyer** [comment]
Would love to see this getting released too

**Comment by qkdreyer** [comment]
@Nyholm @sroze bump 😞

File: Tests/Fixtures/DecoratedPsrMessage.php [hunk 1]
```diff
@@ -15,57 +15,23 @@
...
-     */
-    public function __construct($body = '', array $properties = array(), array $headers = array())
+    public function __construct(string $body = '', array $properties = array(), array $headers = array())
     {
         $this->body = $body;
         $this->properties = $properties;
         $this->headers = $headers;
 
         $this->redelivered = false;
-        $this->flags = self::FLAG_NOPARAM;
```
Comment (sroze) [inline Tests/Fixtures/DecoratedPsrMessage.php:68]: Thanks for the cleanup!

**Review by sroze — APPROVED** [review]

**Pull request merged**